keyboards/ivy/rev1/rev1.h

```diff
@@ -0,0 +1,22 @@
+#ifndef REV1_H
+#define REV1_H
+
+#include "../ivy.h"
+
+//void promicro_bootloader_jmp(bool program);
+#include "quantum.h"
```

Comment:
This isn't needed, since `ivy.h`has it included.  Though, it does no harm here, either. 

```suggestion
```